docs/CLI/index.md

```diff
@@ -50,7 +50,7 @@ set PATH=%USERPROFILE%\bin\;%PATH%
 
 A public Docker image containing the CLI is available on [GitHub Container Registry](https://github.com/orgs/oras-project/packages/container/package/oras):
 
-```
+```bash
```

Comment:
Hmm it’s definitely arduous to have to manually update the latest version in multiple places (the container tag here being one). That’s historically a drawback of mkdocs (over jekyll) that doesn’t support variables in the markdown. Perhaps looking into a plug-in might be an intermediate solution? https://github.com/rosscdh/mkdocs-markdownextradata-plugin

Reply:
@vsoch Good idea! I think we can raise another PR to introduce this plugin and then update those places in the whole documentation. I created a separate [issue](https://github.com/oras-project/oras-www/issues/47) to track it.

Reply:
Sounds good.